src/index.ts

```diff
@@ -14,7 +14,7 @@ export interface VueGtmUseOptions extends GtmSupportOptions {
   /**
    * Don't trigger events for specified router names (case insensitive).
    */
-  ignoredViews?: string[];
+  ignoredViews?: (string | RegExp | ((route: RouteLocationNormalized) => boolean))[];
```

Comment:
Now you overcomplexify some things
See my comment: https://github.com/gtm-support/vue-gtm/pull/37/files#r663366824

An array of functions doesn't even make sense, due to the function can handle what ever it wants already
Also the regex is not needed anymore, cause this can also be handled with the function
And the string array is the normal feature that is already there before this PR

Reply:
sorry about this, I made another commit to fix it just now!

Reply:
Yeah have seen it, okay just finish the PR, I will review it later this evening again

Reply:
Thanks!